lib/autocomplete-manager.coffee

```diff
@@ -188,66 +116,46 @@ class AutocompleteManager
 
     # No suggestions? Cancel autocompletion.
     return unless suggestions.length
+    @suggestionList?.changeItems(suggestions)
+    @showSuggestionList()
+
+  providersForScopes: (scopes) =>
+    return [] unless scopes?
+    return [] unless @scopes
```

Comment:
I know this was here before, but still wondering: Will this ever be `undefined`?